lib/options.js

```diff
@@ -261,7 +261,7 @@ module.exports = function(usingFlatConfig) {
             {
                 option: "ignore-pattern",
                 type: "[String]",
-                description: "Patterns of files to ignore",
+                description: "Patterns of files to ignore. In eslintrc mode, these are in addition to `.eslintignore`",
```

Comment:
Instead of always displaying this, can we use the `usingFlatConfig` variable to only show "these are in addition to .eslintignore" when using eslintrc?

Reply:
I've changed it so the original text will be printed in eslintrc mode. In flat config mode, the notice about `.eslintignore` will be omitted.